### DISCUSSION TITLE
BitmapSource confusion.

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [amerkoleci/Vortice.Windows#347] [Q&A] [answered]
In SharpDX BitmapSource is in WIC, but Vortice has a BitmapSource in Direct2d1.Effects and not in WIC. What's the deal with BitmapSource?

**Comment by amerkoleci** [comment, marked as answer]
Vortice is following native code conventions, not SharpDX, please keep this in mind, vortice exposes IWICBitmapSource and ID2D1BitmapSource.
